packages/cli/globalSetup.js

```diff
@@ -56,7 +56,7 @@ export default async function globalSetup() {
   await fs.writeJson(CONFIG_PATH, TEST_DAEMON_CONFIG)
 
   globalThis.servers = await setup({
-    command: `IPFS_PATH=\'${GOIPFS_DIR_PATH.pathname}\' pnpm dlx @ceramicnetwork/cli daemon --config ${CONFIG_PATH.pathname}`,
+    command: `IPFS_PATH=\'${GOIPFS_DIR_PATH.pathname}\' pnpm dlx @ceramicnetwork/cli@2.35.0-rc.0 daemon --config ${CONFIG_PATH.pathname}`,
```

Comment:
woah, seems weird and dangerous to hard code a specific version here like this.  At least it should probably be `@next` although even that feels a little sketch

Reply:
For this, we specifically want that RC, because otherwise things don't work. It's something that should be removed when we go from RC to release.

Reply:
We want *at least* that RC.  No reason we couldn't take the full release once that RC is promoted to full release

Reply:
For this RC we want _exactly that_ RC. That way if a new RC is released for ceramic, we can validate it, before we push a new RC of js-composedb. This is the same logic as the dependencies. An RC for ceramic could come out that the RC of js-composedb cannot automatically handle. All RC dependencies in js-composedb are adjusted once we determine they should be pulled into js-composedb.

Reply:
https://www.notion.so/threebox/Query-Filters-Query-Ordering-01e9acb5d8b547b89d0ee43dacca3007?pvs=4#0f9ad767d5234fc7bef9c0af01cc1642

Reply:
That is a pretty major change in the way we have always managed the dependency between js-ceramic and js-composedb.  It also seems very backwards to me.  If we do a new RC of js-ceramic, it's most likely because of something we found in testing and is likely something we would want to pick up and include in the next release of js-composedb.  Pinning ourselves to an older RC just adds friction to our testing process and makes it take longer to start getting feedback on if our upcoming release of js-ceramic has problems that would break js-composedb.  If the new RC of js-ceramic is broken we want to learn that ASAP.

Reply:
added https://linear.app/3boxlabs/issue/CDB-2618/determine-appropriate-strategy-for-using-js-ceramic-rcs-in-js